### PR TITLE
Show copied to clipboard when you click the link to copy a story

### PR DESCRIPTION
**Pull request by FionaDL**
This PR closes https://github.com/fastruby/points/issues/198. 

**Description:**

This PR adds the text "Copy to clipboard" when you hover the mouse over the link icon. It also adds a pop-up so that when you click on the icon it will say "Copied to clipboard" before disappearing again. I was not able to screenshot this since it opens and closes quickly. 

Maybe @fbuys wants to check this out since he mentioned the issue. 

___

I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/pull_request_template.md).

### CODE REVIEW

**Comment by kindoflew** [comment]
since Fiona requested you personally, @fbuys would you like to review this? if you have time of course, if not i can ping someone else!

**Comment by fbuys** [comment]
Certainly, will review asap!

File: app/assets/stylesheets/stories.scss [hunk 1]
```diff
@@ -84,3 +84,17 @@
...
+  display: none;
+  position: absolute;
+  top: 5px;
+  left: 30px;
+  background: #d9f1f1;
+  height: 30px;
+  width: 140px;
+  border: 1px solid #9b054d;
+  padding: 5px;
+  align-items: center;
```
Comment (fbuys) [inline app/assets/stylesheets/stories.scss:98]: I though `align-items` and `justify-content` only work in flexboxes?

File: app/views/projects/show.html.erb [hunk 2]
```diff
@@ -22,15 +22,14 @@
...
-
         <tbody id="stories" data-url="<%= sort_stories_project_path(@project.id) %>">
           <% if @stories.present? %>
             <% @stories.each do | story | %>
               <tr class="project-table__row project-table__row--story" id="<%= dom_id(story)%>" >
                 <td class="project-table__cell">
                   <input type="checkbox" name="stories[]" value=<%= story.id %>>
                   <%= link_to story.title, [story.project, story] %>
-                  <button class="copy-link btn-clipboard" data-clipboard-text="<%= project_story_url(@project, story) %>"><i class="fa fa-link"></i></button>
+                  <button class="copy-link btn-clipboard" data-clipboard-text="<%= project_story_url(@project, story) %>" title='Copy to clipboard'><i class="fa fa-link"></i><span class= "popup">Copied to clipboard</span></button>
```
Comment (fbuys) [inline app/views/projects/show.html.erb:32]: How do you feel about making the title `Click to copy`?

**Review by fbuys — COMMENTED** [review]
Looks great! 🚀
I made some comments, let me know what you think.

File: app/assets/stylesheets/stories.scss (hunk 1 above)
Comment (FionaDL) [reply app/assets/stylesheets/stories.scss:98]: I think they can work with grid too, but you are right, I had this leftover.

**Comment by FionaDL** [comment]
@fbuys I think I have addressed your feedback :)

**Comment by lubc** [comment]
@mateusdeap Can you QA this PR? Thanks!

**Review by bronzdoc — APPROVED** [review]

**Comment by bronzdoc** [comment]
I QAd this, Looks good 👍

**Pull request merged**